src/frontend-samples/viewport-only-sample/ViewportOnlyUI.tsx

```diff
@@ -2,23 +2,66 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-import * as React from "react";
```

Comment:
@Josh-Schifter Can you comment on this sample code?

Reply:
It is a lot more code than before.  Was 24 lines, now 67 lines.  That is unfortunate but I don't see an obvious way to condense it.